src/Plugin/Condition/DataComparison.php

```diff
@@ -24,7 +24,8 @@
  *     "operator" = @ContextDefinition("string",
  *       label = @Translation("Operator"),
  *       description = @Translation("The comparison operator."),
- *       required = FALSE
+ *       default_value = "==",
+ *       required = TRUE
```

Comment:
@klausi -- I think that this was done due to -[Context in Rules plugins does not respect annotation order](https://www.drupal.org/node/2573823)-.  Until that issue was fixed, unless it was marked "required",  `required = TRUE` was needed for the default value to get used, oddly enough.

I've just done some tests, and this is no longer an issue.  I'll change this back to `required = FALSE`.